lib/pkg.ml

```diff
@@ -515,21 +515,30 @@ let assert_tag_exists ~dry_run repo tag =
   if Vcs.tag_exists ~dry_run repo tag then Ok ()
   else R.error_msgf "%s is not a valid tag" tag
 
-let distrib_archive ~dry_run ~keep_dir p =
+let pull_submodules ~dry_run ~dist_build_dir =
+  Sos.with_dir ~dry_run dist_build_dir
+    (fun () -> Vcs.get () >>= Vcs.submodule_update ~dry_run)
+    ()
+  |> R.join
```

Comment:
as it doesn't involve a `Pkg.t` type wouldn't it better to put this function in vcs.ml ?